frontend/src/app/home/components/SelfReview.tsx

```
@@ -20,11 +20,13 @@ interface SelfReviewProps {
   onReviewResultsReceived: (
     self_review: Array<{ question: string; answer: string }>
   ) => void;
+  setIsReviewGenerating: (value: Boolean) => void;
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)